buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -55,6 +55,9 @@ class ClusterFormationTasks {
      * Returns a list of NodeInfo objects for each node in the cluster.
      */
     static List<NodeInfo> setup(Project project, String prefix, Task runner, ClusterConfiguration config) {
+        if ("false".equals(System.getProperty("tests.allow.ClusterFormationTasks", "true"))) {
+            throw new IllegalStateException("Using cluster formation tasks but these are not allowed in this build");
```

Comment:
If we do this, we should give folks better instruction. Even so far as to say "Ping Alpar on Slack for more info" :wink:

At the very least we should point folks towards `TestClusters`.